.github/ISSUE_TEMPLATE/design_call.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Community or working group call
+about: Bitcoin Design community call or design sessions
+title: '... Design Call #0'
+labels: 'call'
```

Comment:
Are those labels automatically added to the issue somehow? Or just for individual reference?

Reply:
The "call" label will automatically get added to any new issue created with this template. Currently I go through and manually tag call issues cause I think only maintainers can.

Maybe this can be abused in the future, if we have any external notifications tied to these labels.

Reply:
Just to clarify, does the "call" label get added because it is added here in this template, or because of some other automation mechanism?

Reply:
Yes, it will be added because it's set in this template file. Nothing external.